docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -110,11 +108,31 @@ The following decision tree shows all of the paths from delivery facility choice
 
 where :math:`p(\text{RDS})` is the probability of dying from Preterm with RDS in the general population, and :math:`p(\text{RDS}|\text{CPAP})` and :math:`p(\text{RDS}|\text{no CPAP})` are the probability of dying from Preterm with RDS in setting with and without access to CPAP.  For each path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges labeled BEmONC and unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has CPAP available (7.5% in Ethiopia in 2016)
 
-When we fill in the location-specific values for delivery facility rates, CPAP coverage, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_RDS_w and p_RDS_wo), which we can solve analytically.
+When we fill in the location-specific values for delivery facility rates, CPAP coverage by facility type, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (p_RDS_w and p_RDS_wo), which we can solve analytically.
+
+As mentioned above, it is convenient to model this intervention like a dichotomous risk factor, so that we can reuse the
+:class:`Risk<vivarium_public_health.risks.base_risk.Risk>`
+and :class:`RiskEffect<vivarium_public_health.risks.effect.RiskEffect>` components in Vivarium Public Health,
+rather than having to write new components from scratch.
+Calling CPAP a risk factor can sound a bit confusing because CPAP access is a good thing, so it doesn't sound "risky."
+Instead, we flip it so the risk factor is "*lack* of access to CPAP."
+
+The :code:`RiskEffect` component expects a relative risk (RR) and a population-attributable fraction (PAF).
+Because we are flipping the direction of the risk factor, we need to use the inverse of our original RR, so:
+
+.. math::
+    \text{RR}_{\text{no CPAP}} = \frac{1}{\text{RR}_{\text{CPAP}}}
+
+The PAF is the proportion of deaths due to preterm with RDS that would not occur if all births had access to CPAP.
+Since we use the equation :math:`p(\text{RDS}|\text{CPAP}) = (1 - \text{PAF}_\text{no CPAP}) \cdot p(\text{RDS})`
```

Comment:
I'm not familiar with the code in the `RiskEffect` component, but based on how this intervention should work, where would this equation get used?

I think the simulation is never using the probabilities $p(\text{RDS}|\text{CPAP})$ and $p(\text{RDS})$ directly, because it is operating on CSMRs, not probabilities. I.e., the "treatment-deleted" $\text{CSMR}\_i^\text{RDS}$ would be $(1 - \text{PAF})\cdot \text{CSMR}\_i^\text{RDS}$, which then gets multiplied by the simulant's individual RR. It would be interesting to explain why the PAF for the CSMRs should be the same as the PAF for the probabilities, which is not immediately obvious to me...

Reply:
You're right and I guess I was hoping to get away with this imprecision. Once we have `p = r * dt` in Vivarium, the PAFs will be the same. But right now they (slightly) aren't, but should be really close if our timesteps are short

Reply:
This simulation has no timesteps... we're using $p(\text{neonate dies}) = 1 - \exp(-\text{ACMR} \cdot \Delta t)$, where $\Delta t$ is either 7 or 21 days.

Reply:
Oh right! Yeah, in that case it does technically matter. I suppose we could change this whole page to deal in rates instead of probabilities? Or is there a simpler path forward?

Reply:
> You're right and I guess I was hoping to get away with this imprecision.

I think you can get away with the imprecision -- maybe just don't say the `RiskEffect` component uses that equation, since I don't think that's true in this case, which could be confusing.